lib/adding_columns.py

```diff
@@ -91,13 +98,23 @@ def add_average_films_ratings(spark: SparkSession, data: DataFrame) -> DataFrame
         .select(["nconst", "average_films_rating"])
     )
     data = data.join(data_with_ratings, on="nconst", how="left")
+    data = data.na.fill(value = 0, subset = ["average_films_rating"])
     return data
 
+def add_normalized_columns(data: DataFrame) -> DataFrame:
+    unlist = udf(lambda x: round(float(list(x)[0]),3), DoubleType())
```

Comment:
proszę o asercje na brak nowotworzonych kolumn w wejściowycm datasetcie

Reply:
dodane - nie wiem czy dokładnie o to chodziło

Reply:
nie widzę, żadnych asercji

Reply:
w takim razie potrzebuję doprecyzowania, co dokładnie jest oczekiwane - dodałam if sprawdzający czy kolumny już nie istnieją w dataframie, jeśli istnieją, wyrzucany jest błąd.

Reply:
chyba już zrozumiałam, czy o to chodziło?

Reply:
tak, dokładnie o to